examples/new_example.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+// require 'vendor/autoload.php'; // Make sure to include Composer's autoload file
+require '../init.php';
+
+class new_example
```

Comment:
Similarly `NewExample.php` and
```suggestion
class NewExample
```